great_expectations/zep/interfaces.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Protocol, Union
```

Comment:
Missed this initially, but until we get off of `python 3.7` we'll need to do something like this...

https://stackoverflow.com/a/71995876/6304433

Reply:
Thanks for catching this!
Yeah, we don't need the `else` logic. We actually use `typing_extensions` here:
https://github.com/great-expectations/great_expectations/blob/213b5609633a38857ef2cf9f89d2fe7a4112c7dd/tests/integration/profiling/rule_based_profiler/test_profiler_user_workflows.py#L16

Reply:
Done.